### PR TITLE
MC Bundles static and dynamic project config issue fixes

### PR DESCRIPTION
**Pull request by praveenkumarct**
**Bundles static and dynamic project config level issues fixes and refactoring:**
- Remove usage of `mc-url` and use `window.location.origin`.
- Remove usage of headers and env files for the deployment of Firebase and AWS.
- Add env files for all levels of deployments and local. Placeholders in custom config file. Check [docs](https://docs.commercetools.com/custom-applications/api-reference/cli#using-dotenv-files) for more details.
- Remove Transformers for Firebase and AWS deployments and add separate Json files for basic setup.
- Remove eslint and babel and use existing from MC frontend libraries.
- Specify the target under context, while running graphql queries. Check [here](https://docs.commercetools.com/custom-applications/development/data-fetching) for more details.
- Update doc.

Related issues:
- https://github.com/commercetools/commercetools-bundles-starter/issues/223
- https://github.com/commercetools/commercetools-bundles-starter/issues/222
- https://github.com/commercetools/commercetools-bundles-starter/issues/221

### CODE REVIEW

File: packages/bundles-dynamic/aws.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (emmenko) [inline packages/bundles-dynamic/aws.json:1]: Is the filename correct? `aws.json`

File: packages/bundles-dynamic/src/components/bundle-prices/bundle-prices.js [hunk 2]
```diff
@@ -16,16 +16,15 @@ import messages from './messages';
...
+  const { project } = useApplicationContext();
   const { currencies } = project;
   const [currency, setCurrency] = useState(currencies[0]);
   const [country, setCountry] = useState(null);
   const [customerGroup, setCustomerGroup] = useState(null);
   const [channel, setChannel] = useState(null);
   const [date, setDate] = useState('');
 
-  const MC_URL = `https://${mcURL}/${match.params.projectKey}`;
+  const MC_URL = `${window.location.origin}/${match.params.projectKey}`;
```
Comment (emmenko) [inline packages/bundles-dynamic/src/components/bundle-prices/bundle-prices.js:27]: Do you plan to do the refactoring of the routing as a follow up PR? I'm referring to removing the usage of `window.open`.

**Review by emmenko — COMMENTED** [review]
Had a quick look, looks good so far!

File: packages/bundles-dynamic/src/components/bundle-prices/bundle-prices.js (hunk 2 above)
Comment (praveenkumarct) [reply packages/bundles-dynamic/src/components/bundle-prices/bundle-prices.js:27]: Yes, that's the plan. Created sprint ticket in backlog: https://jira.commercetools.com/browse/PSS-515

File: packages/bundles-dynamic/aws.json (hunk 1 above)
Comment (praveenkumarct) [reply packages/bundles-dynamic/aws.json:1]: I couldn't find any proper information for this deployment json file usage [here](https://docs.commercetools.com/custom-applications/deployment-examples/aws-s3-cloudfront).
Can you guide me to the docs which helps me understand how this deployment works?

**Review by nspeet — APPROVED** [review]

File: packages/bundles-dynamic/aws.json (hunk 1 above)
Comment (praveenkumarct) [reply packages/bundles-dynamic/aws.json]: Thank you @emmenko.

@nspeet I see there is [doc](https://github.com/commercetools/commercetools-bundles-starter/tree/master/packages/bundles-static#deployment) on deployment for Firebase and AWS, which won't work properly after these PR changes. 
And the links in the document is already broken, it is not found anymore from Commercetools documents. Ex: [Firebase](https://docs.commercetools.com/custom-applications/deployment/example-deployment-firebase)
[AWS](https://docs.commercetools.com/custom-applications/deployment/example-aws-s3-cloudfront)

Do you think we need to setup and provide a proper documentation from our end(Only for Bundles-static and dynamic projects) to perform deployment for both Firebase and AWS. Or this should be handled by users and we will provide support only for the deployment options available in MC custom applications [documentation](https://docs.commercetools.com/custom-applications/deployment-examples)?

Comment (nspeet) [reply packages/bundles-dynamic/aws.json]: Hi @praveenkumarct, in the bundles doc let's link to the MC app deployment options latest [doc](https://docs.commercetools.com/custom-applications/deployment-examples)

Comment (praveenkumarct) [reply packages/bundles-dynamic/aws.json]: https://github.com/commercetools/commercetools-bundles-starter/pull/231/commits/0588eb2944af98c1e431f43af46a3601a48185e0

As discussed, I have removed the legacy config for AWS and Firebase deployments available in MC bundles project and pointed the users to the MC official documentation for deployment examples with some hints.

**Pull request merged**